kuroko2.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'retryable'
   s.add_dependency 'faraday'
   s.add_dependency 'sprockets'
+  s.add_dependency 'slim'
```

Comment:
It is for preventing https://travis-ci.org/cookpad/kuroko2/jobs/225911655 .
But it seems it was another cause.  (Maybe the responders gem update was effective )
So I revert this changes 3eb2aa3 .